site/src/components/example-container/ExampleConainer.module.css

```diff
@@ -0,0 +1,66 @@
+.root {
```

Comment:
file name: 
+ ExampleContainer.module.css
- ExampleConainer.module.css

Reply:
Good spot! Will push a fix soon

Reply:
Ok fixed it now. Could you please re-approve the PR if you're happy with it?

Reply:
Thanks!